solidity/test/VendingMachineTest.js

```diff
@@ -531,6 +531,29 @@ describe("VendingMachine", async function() {
           "Bad _extraData signature. Call must be to tbtcToBtc.",
         )
       })
+
+      it("protects receiveApproval from external callers", async () => {
+        const tbtcToBtc = vendingMachine.abi.filter(
+          x => x.name == "tbtcToBtc",
+        )[0]
```

Comment:
This can just be `find(...)` instead of `filter(...)[0]`.

Reply:
indeed